src/AspNet.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -9,18 +9,41 @@ public static class OpenIdConnectConstants {
         public static class Claims {
             public const string Active = "active";
             public const string Address = "address";
+            public const string AtHash = "at_hash";
+            public const string Birthdate = "birthdate";
             public const string ClientId = "client_id";
             public const string Confidential = "confidential";
+            public const string Email = "email";
             public const string EmailVerified = "email_verified";
+            public const string FamilyName = "family_name";
+            public const string Gender = "gender";
+            public const string GivenName = "given_name";
+            public const string Locale = "locale";
+            public const string Name = "name";
+            public const string Nickname = "nickname";
+            public const string MiddleName = "middle_name";
             public const string PhoneNumber = "phone_number";
             public const string PhoneNumberVerified = "phone_number_verified";
+            public const string Picture = "picture";
             public const string PreferredUsername = "preferred_username";
             public const string Profile = "profile";
             public const string Scope = "scope";
+            public const string Subject = "sub";
             public const string TokenType = "token_type";
+            public const string UpdatedAt = "updated_at";
             public const string Usage = "usage";
             public const string Username = "username";
             public const string Website = "website";
+            public const string Zoneinfo = "zoneinfo";
+
+            public static class AddressClaims {
```

Comment:
Yeah, I think that would be enough. Any more grouping proposals @MonkeyJamboree?